concepts/constants/introduction.md

```diff
@@ -25,5 +25,5 @@ const (
     g
 )
 fmt.Print(a, b, c, d, e, f, g)
-// Output: 9 9 9 3 4 5 6
+// => 9 9 9 3 4 5 6
```

Comment:
Since this is an instance where print functions are used, we should keep the `Output:` wording.